pkg/metrics/metrics.scss

```diff
@@ -498,13 +498,18 @@ html {
       --color: lightsalmon;
     }
 
+    .graph-over {
+      fill: red;
```

Comment:
I find this really hurtful to the eyes. Could we change this to be a little less intense?